src/components/user/MenuLeft.js

```diff
@@ -0,0 +1,51 @@
+import React, { PureComponent } from 'react';
+import { array, func, object, string } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+import LogoutButton from 'components/user/LogoutButton';
+import Welcome from 'components/user/Welcome';
+import CreateTargetForm from 'components/user/CreateTargetForm';
+import AboutTarget from 'components/user/AboutTarget';
+import routes from 'constants/routesPaths';
+
+export default class MenuLeft extends PureComponent {
+  static propTypes = {
+    title: string,
+    topicList: array,
+    section: string,
+    handleCreateTarget: func,
```

Comment:
This function should be required, right?

Reply:
In this case, the children component is which using this function and has a prop required. I added this in any way.